opm/material/fluidmatrixinteractions/PiecewiseLinearTwoPhaseMaterialParams.hpp

```diff
@@ -180,10 +217,7 @@ class PiecewiseLinearTwoPhaseMaterialParams : public EnsureFinalized
     void swapOrder_(ValueVector& swValues, ValueVector& values) const
     {
         if (swValues.front() > values.back()) {
```

Comment:
This line is a clear and old bug, is it not?!?

Reply:
This does look strange to me too, yet the line is 9 years old! Do you also think it should be swValuas.back() instead of values.back(), becaues the saturation values are what we binary search over assuming low-high order?

Reply:
Yes, that is what I think. Changing it to swValues.back() should be done.

Reply:
"Fixing" this line by making this adjustment breaks tests. Both the hysteresis test in opm-common, but also many compareECLFile tests in simulators. Should we for now leave a TODO here to bring attention to this odd-looking line and not break the tests?

Reply:
we should at least do it in a separate PR.

Reply:
no, a separate PR fixing the bug.